src/reportWebVitals.js

```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
```

Comment:
Are you using this? if yes do you really require this

Please remove this file if possible